ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/TransactionCommitter.scala

```diff
@@ -249,20 +249,11 @@ private[kvutils] class TransactionCommitter(
       })
 
   /** Validate the submission's conformance to the DAML model */
-  private def authorizeAndBlind: Step =
-    (commitContext, transactionEntry) =>
-      Blinding
-        .checkAuthorizationAndBlind(
-          transactionEntry.transaction,
-          initialAuthorizers = Set(transactionEntry.submitter),
-        )
-        .fold(
-          error =>
-            reject(
-              commitContext.getRecordTime,
-              buildRejectionLogEntry(transactionEntry, RejectionReason.Disputed(error.msg))),
-          blindingInfo => buildFinalResult(commitContext, transactionEntry, blindingInfo)
-      )
+  private def authorizeAndBlind: Step = //TODO: rename. no authorization here any more!
```

Comment:
Why not renaming now?

Reply:
sure
the only reason for stopping was I didn't know how far the cascade of changes would go.

Reply:
You just need to change this line together with the line 70.

```scala 
"authorize_and_blind" -> authorizeAndBlind
```
to 
```scala 
"blind" -> blind
```

Reply:
@miklos-da or @SamirTalwar could you confirm ?

Reply:
Yeah, this rename should be pretty benign.

Reply:
will do it